src/AzureAuth/Ado/Constants.cs

```diff
@@ -8,6 +8,11 @@ namespace Microsoft.Authentication.AzureAuth.Ado
     /// </summary>
     internal static class Constants
     {
+        /// <summary>
+        /// ADO pipeline id.
+        /// </summary>
+        public const string SystemDefinitionId = "SYSTEM_DEFINITIONID";

```

Comment:
Let's add the doc - https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services

Reply:
Yeah we can do that.